db/migrate/20180731142628_add_column_comment_to_qc_result.rb

```diff
@@ -1,5 +1,5 @@
 class AddColumnCommentToQcResult < ActiveRecord::Migration
   def change
-    set_column_comment :qc_result, :last_updated, 'The date the qc_result was last updated in SS'
+    change_column_comment :qc_result, :last_updated, from: nil, to: 'The date the qc_result was last updated in SS'
```

Comment:
In practice we're unlikely to ever run this again, but this updates it from using the migration_comments syntax to the native rails syntax. migration_comments is abandoned and was blocking the Ruby 3 update.